src/App.js

```diff
@@ -1,5 +1,17 @@
 import React from "react";
+import {Routes, Route} from "react-router-dom";
+import HomePage from "../src/components/HomePage";
+import  BookingSeat from "../src/components/BookingSeat";
+function App() {
+  return (
+    <>
+      <Routes>
+  <Route path="/" element={<HomePage/>}/>
+  <Route path="/bookingseat/:id/:path" element={<BookingSeat/>}/>
```

Comment:
The redirected url when a movie is clicked - `http://localhost:3000/bookingseat/438148/wKiOkZTN9lUUUNZLmtnwubZYONg.jpg` The url should be - `minions-the-rise-of-gru`